### ISSUE TITLE
Remove Anonymization Functionality from plugin

### ISSUE THREAD

**Issue by NickGreen** [a8cteam51/safety-net#55]
This is very cool functionality, but due to the limitations that we have from systems, we should probably remove this from the plugin. That way, we won't get someone trying to run it on a large site and getting a ping from systems again.

Also, removing references to it in the README.

**Comment by NickGreen** [comment]
Removed Anonymization from admin in #62 

As for having it in the plugin _somewhere_, it might be useful to keep it as a CLI command in case we want to run it locally, for example. TBD

**Comment by bernattorras** [comment]
I agree that keeping this option in the CLI could be useful. 

What about adding a warning message prior to the execution of the anonymize() function to warn the user about the risks of this command and only proceed if they agree?

Since these processes can take a bit of time, I think it could also be useful to provide some feedback in the terminal to show at least the action that is currently being performed (i.e: "`1/3 - Anonymizing users ...`").

**Comment by bernattorras** [comment]
I've submitted the PR #77 because I've found that the wp `safety-net anonymize` CLI command wasn't working properly (it was stuck because the temporary tables were only created for the background processes).

I've also:
- Removed an orphan `wp_ajax_safety_net_anonymize_users hook` from the admin.php 
- Displayed a warning and asked for a confirmation before starting the anonymization process. 
- Added some feedback lines during the execution of the different anonymization functions.

We should be able to close this issue if the PR #77 gets merged.
